receiptrepository/src/main/java/com/hyperwallet/android/ui/receipt/repository/PrepaidCardReceiptDataSourceFactory.java

```diff
@@ -27,10 +27,12 @@
 public class PrepaidCardReceiptDataSourceFactory extends DataSource.Factory {
 
     private final MutableLiveData<PrepaidCardReceiptDataSource> mDataSourceMutableLiveData;
-    private final PrepaidCardReceiptDataSource mPrepaidCardReceiptDataSource;
+    private PrepaidCardReceiptDataSource mPrepaidCardReceiptDataSource;
+    private final String token;
```

Comment:
I think we are not using this variable anywhere. Am i right/

Reply:
Sorry, peter above comments. I think we need to use prefix to the variable.

Reply:
oh yeah thanks @manikandan-altmtrk 👍